### ISSUE TITLE
unable to convert some of the lambda cap assets

### ISSUE THREAD

**Issue by shinjantiwary** [skynav/cap2tt#2]
I am using the following command to convert the assets located in: "CAP Test Assets"
-d --warn-on all --force-encoding Shift_JIS --output-indent --output-directory <outDir> <input.cap>

Conversion of the following assets failed with following errors:

Error: Unmappable Shift_JIS at byte offset 0 of 2 bytes.
Assets: PonyCanyon_NPV_Origination__.cap

Error: [1,0]:Header bad field count, got 1, expected 3.
Assets: SonyPCL_NPV_Origination__.cap

Error: Unmappable Shift_JIS at byte offset 38 of 2 bytes
Asset: VideoTech_NPV_Origination_FULL.cap

Could I be missing some options that would help us convert them successfully?

**Comment by skynavga** [comment]
The first and third files (PonyCanyon and VideoTech) fail because they do not use Shift JIS, but rather use UTF-8. However, simply changing the command option to UTF-8 doesn't work because UTF-8 is not yet declared as a permitted encoding. Also, even if that is fixed, the second file, VideoTech, fails because it has a BOM and there appears to be a bug causing BOM not to be skipped before parsing header line. I have created issue #5 and issue #6 to address these problems.

The second file (SonyPCL) fails because its header line is missing the DF\* and Standard Scene fields. I have created issue #7 to address this problem.
